Assets/UI/worldviewiconsmanager_CQUI.lua

```diff
@@ -7,27 +7,32 @@ include("WorldViewIconsManager");
 -- Cached Base Functions
 -- ===========================================================================
 BASE_SetResourceIcon = SetResourceIcon;
+BASE_AddImprovementRecommendationsForCity = AddImprovementRecommendationsForCity;
 
 -- ===========================================================================
 -- CQUI Members
 -- ===========================================================================
-local CQUI_ResourceIconStyle = 1;
+local CQUI_ResourceIconStyle:number = 1;
+local CQUI_ShowImprovementsRecommendations:boolean = false;
 
 function CQUI_GetSettingsValues()
-  CQUI_ResourceIconStyle = GameConfiguration.GetValue("CQUI_ResourceDimmingStyle");
-  if CQUI_ResourceIconStyle == nil then
-    print("CQUI_ResourceIconStyle is nil!  Using default value.");
-    CQUI_ResourceIconStyle = 1;
-  end
+	CQUI_ResourceIconStyle = GameConfiguration.GetValue("CQUI_ResourceDimmingStyle");
+	if CQUI_ResourceIconStyle == nil then
+		print("CQUI_ResourceIconStyle is nil!  Using default value.");
+		CQUI_ResourceIconStyle = 1;
+	end
```

Comment:
code style:  after an "end" I have been putting a blank line, unless it's immediately followed by an else(if) or another end
```suggestion

  end
```

Reply:
@the-m4a I think your suggestion is using spaces instead of the tab character that Infixo seems to be using. Part of the reason I keep harping on about consistency. Also you suggestion looks like the empty line is before the end and not after.

Reply:
tried making the suggestion on the site, which I don't think let me insert the tab anyway... and yes - the blank line should've been after.

Reply:
I edited the suggested fix, including manually pasting the tab character